Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -170,6 +172,29 @@ def get_geo_df(data: pd.DataFrame,
     return output
 
 
+def animate(data: pd.DataFrame, filepath: str, fps: int = 3, dpi: int = 150, **kwargs) -> None:
+    """Generate an animated mp4 file of a signal over time.
+
+    :param data: DataFrame for a single signal over time.
+    :param filepath: Path where video will be saved.
+    :param fps: Frame rate in frames per second for animation. Defaults to 3.
+    :param dpi: Dots per inch for output video. Defaults to 150 on a 12.8x9.6 figure (1920x1440).
```

Comment:
Probably should document `kwargs` to say they go to `plot_choropleth`, e.g. so you can set figsize

Reply:
oops forgot that, added.